### ISSUE TITLE
Pipeline scheduled to run after another pipeline ends is not launched when the first pipeline is launched on demand

### ISSUE THREAD

**Issue by OskarStoffan** [UnifiedViews/Core#432]
When a scheduling rule is set for pipeline1 to run after pipeline2 ends, pipeline1 is not launched when pipeline2 is executed manually on demand. This feature only works if there is a scheduling rule defined which executes the pipeline2.

**Comment by skrchnavy** [comment]
@tomas-knap is this feature that was planed in UV or it is enhancement that is useful and expected or against defined requirements?

**Comment by tomas-knap** [comment]
Yes, you are right. I would suggest to modify the label of the rule below so that it is clear that it runs only if the pipeline after it should be executed was executed by scheduller (and not manually): 

<img width="355" alt="screen shot 2015-08-11 at 12 53 35" src="https://cloud.githubusercontent.com/assets/3014917/9196072/25424144-4028-11e5-94ab-7b6f0f30a5c0.png">

Later on, if this is a blocker for somebody, we may work on the improvement of the current behavior.
